Assets/Scripts/CardsAdder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using System.Collections;
+using System;
+
+public class CardsAdder : MonoBehaviour
+{
+    [SerializeField] private CardInfo[] _cards;
+
+    [SerializeField] private GameObject _cardPrefab;
+    [SerializeField] private Transform _cardsField;
+
+    private int _cardsAmount = 9;
+
+    private float _secondsToAdd = 0.1f;
+
+    public CardInfo[] CardsTemplate { get => _cards; }
+
+    public event Action CardsAdded;
+
+    private void Awake()
+    {
+        StartCoroutine(AddCards());
+    }
+
+    private void AddCardsOnScene(int number)
+    {
+        GameObject card = Instantiate(_cardPrefab);
+
+        card.GetComponent<CardDisplay>().Card = _cards[number];
+        card.GetComponent<CardDisplay>().SetValues();
+        card.transform.SetParent(_cardsField, false);
```

Comment:
https://trello.com/c/orS9j3Rj/6-%D0%B4%D0%B8%D0%BD%D0%B0%D0%BC%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5-%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82%D0%BE%D0%B2